mesonbuild/backend/backends.py

```diff
@@ -1815,10 +1819,10 @@ def generate_target_install(self, d: InstallData) -> None:
                             i = TargetInstallData(f, first_outdir, first_outdir_name,
                                                   False, {}, set(), None, install_mode,
                                                   t.subproject, optional=not t.build_by_default,
-                                                  tag=tag)
+                                                  tag=tag, out_fname=rename[0])
```

Comment:
There is only one outdir, but all outputs are installed in the same outdir? Why would this use `rename[0]` instead of the corresponding item in the array?

Reply:
thanks, I just missed this case when walking through the multiple-output cases.